src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Rectangle extends FigureTwo implements AreaOfFigure, InfoFigure {
+    //прямоугольник
+
+    public Rectangle(String color,double firsSide, double secondSide) {
+        super(color, firsSide, secondSide);
+    }
+
+    @Override
+    public double detAreaOfFigure() {
```

Comment:
There is a typo in the method name `getFirsSide`. It should be `getFirstSide` to match the corrected variable name in the `FigureTwo` class.